docs/cirq_interface.md

```diff
@@ -148,6 +148,6 @@ Known gates with no decomposition:
 ### Parametrized circuits
 
 QSimCircuit objects can also contain
-[parameterized gates](https://cirq.readthedocs.io/en/stable/tutorial.html#parameterizing-the-ansatz)
+[parameterized gates](https://cirq.readthedocs.io/en/stable/docs/tutorials/variational_algorithm.html#Parameterizing-the-Ansatz)
```

Comment:
can you change it to https://cirq.readthedocs.io/en/stable/docs/tutorials/basics.html#Using-parameter-sweeps instead?